### PR TITLE
Update mini-dashboard images

### LINKED ISSUE #1666: Update mini-dashboard image
The `learn/what_is_meilisearch/overview.html` and the `earn/what_is_meilisearch/search_preview.html` pages use an image of an "old" mini-dashboard with the former spelling of Meilisearch
Capture d’écran 2022-05-30 à 14 00 33

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #1666

### CODE REVIEW

**Review by dichotommy — COMMENTED** [review]
LGTM! Would it be possible to scale the images to the previous resolution? Just because it seems useful to hit approximately 1440p.

**Review by CaroFG — DISMISSED** [review]

File: learn/getting_started/quick_start.md [hunk 1]
```diff
@@ -261,10 +261,12 @@ By default, Meilisearch only returns the first 20 results for a search query. Th
 
 Meilisearch offers a search preview where you can preview search results. It comes with a search bar that allows you to search a selected index. You can access it in your browser at `http://127.0.0.1:7700` any time Meilisearch is running.
 
-![Meilisearch's search preview indicating the indexes dropdown on the upper right corner](/getting-started/multiple_indexes.png)
+![Meilisearch's search preview showing the movies index](/search_preview/default_search_preview.png)
 
 If you have multiple indexes, you can switch between them using the indexes dropdown.
 
+![Meilisearch's search preview indicating the indexes dropdown on the upper right corner](/search_preview/multipl_indexes.png)
```
Comment (dichotommy) [inline learn/getting_started/quick_start.md:268]:
```suggestion
![Meilisearch's search preview indicating the indexes dropdown in the upper right corner](/search_preview/multipl_indexes.png)
```

File: learn/what_is_meilisearch/search_preview.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ After adding documents to your Meilisearch instance, you can immediately start s
 
 If your Meilisearch instance does not have any indexes, you should see this screen.
 
-![Screenshot of the search preview instructing the user to set an API key and configure an index](/search_preview_no_documents.png)
+![Meilisearch search preview instructing the user to set an API key and configure an index](/search_preview/search_preview_no_documents.png))
```
Comment (dichotommy) [inline learn/what_is_meilisearch/search_preview.md:7]:
```suggestion
![Meilisearch search preview instructing the user to set an API key and configure an index](/search_preview/search_preview_no_documents.png)
```

**Review by dichotommy — COMMENTED** [review]
LGTM, just one or two comments + some file names that can be improved. 

- `.vuepress/public/search_preview/multipl_indexes.png` missing an `e` at the end of `multiple`
- `.vuepress/public/search_preview/default_search_preview.png` could be renamed just `default`
- `.vuepress/public/search_preview/search_preview_no_documents.png` could be renamed just `no_documents`

**Review by dichotommy — APPROVED** [review]
LGTM! 🥇

**Comment by maryamsulemani97** [comment]
Finally 😭

bors merge

**Pull request merged**